packages/components/src/DataList/components/DataListHeaderTile/DataListSortingArrows.css

```diff
@@ -16,3 +16,8 @@
 .sortIcon path.inactive {
   fill: var(--color-inactive--surface);
 }
+
+.sortIcon:hover path {
+  filter: brightness(85%);
```

Comment:
Let's use `var(--color-interactive--subtle)` here; our preference for hover interactions is to darken, and to use set values rather than filters/tints/opacities

Reply:
Also I lied, one other small change - can we match the alignment of the right-aligned sort icon to the column's contents? Reference [here](https://www.figma.com/file/TuoXq9l7Mg1Xfusj6qnx31/%5BJobber-2025-System%5D-List-Views?type=design&node-id=4514%3A63926&mode=design&t=m6hsxzxVr8mU9hRr-1)
![image](https://github.com/GetJobber/atlantis/assets/39704901/5c4e2c99-3248-438f-a74c-e17e1242fa1f)

Reply:
(If the alignment is proving finicky to resolve, the color change would be sufficient within the scope of this PR)

Reply:
@scotttjob can attest I really tried to fix that alignment, but ultimately failed when the clock ran out. Ended up deciding to just get this in for now. The tip on `var(--color-interactive--subtle)` is appreciated, will push that change up soon.